src/stores/widgetManager.ts

```diff
@@ -60,6 +65,167 @@ export const useWidgetManagerStore = defineStore('widget-manager', () => {
   )
   const _widgetManagerVars: Ref<Record<string, WidgetManagerVars>> = ref({})
   const _miniWidgetManagerVars: Ref<Record<string, MiniWidgetManagerVars>> = ref({})
+  const isElementsPropsDrawerVisible = ref(false)
+  const elementToShowOnDrawer = ref<CustomWidgetElement>()
+  const widgetToEdit = ref<Widget>()
+  const miniWidgetLastValues = useBlueOsStorage<Record<string, any>>('mini-widget-last-values', {})
```

Comment:
One small thing here: the storage key should start with "cockpit-...".

Reply:
I also have relocated the save widget method to outside the store